instrumentation/messaging/RATIONALE.md

```diff
@@ -78,3 +78,155 @@ continue incoming state.
+| JMS         | PRODUCER | Send           | Response  | JMSMessageId                | Remote | Global     | ID:opaque string
+| JMS         | CONSUMER | Receive        | Request   | JMSMessageId                | Remote | Global     | ID:opaque string
+| MQTT        | PRODUCER | PUBLISH        | Request   | Packet Identifier           | Local  | Connection | uint16
+| MQTT        | PRODUCER | PUBACK/PUBREC  | Response  | Packet Identifier           | Local  | Connection | uint16
+| MQTT        | CONSUMER | PUBLISH        | Request   | Packet Identifier           | Remote | Connection | uint16
+| MQTT        | CONSUMER | PUBACK/PUBREC  | Response  | Packet Identifier           | Remote | Connection | uint16
+| Pulsar      | PRODUCER | Send           | Response  | MessageId                   | Remote | Topic      | bytes(ledger|entry|parition)
+| Pulsar      | CONSUMER | Receive        | Request   | MessageId                   | Remote | Topic      | bytes(ledger|entry|parition)
+| RocketMQ    | PRODUCER | send           | Response  | msgId                       | Remote | Topic      | HEX(ip|port|offset)
+| RocketMQ    | CONSUMER | consumeMessage | Request   | msgId                       | Remote | Topic      | HEX(ip|port|offset)
+| ServiceBus  | PRODUCER | POST           | Request   | BrokerProperties{MessageId} | Local  | Global     | 1-255 characters
+| ServiceBus  | CONSUMER | DELETE         | Request   | BrokerProperties{MessageId} | Remote | Global     | 1-255 characters
+| STOMP       | PRODUCER | SEND           | Request   | receipt Header              | Local  | Connection | arbitrary
+| STOMP       | PRODUCER | RECEIPT/ERROR  | Response  | receipt-id Header           | Local  | Connection | arbitrary
+| STOMP       | CONSUMER | SEND           | Request   | receipt Header              | Remote | Connection | arbitrary
+| STOMP       | CONSUMER | RECEIPT/ERROR  | Response  | receipt-id Header           | Remote | Connection | arbitrary
+| STOMP       | PRODUCER | MESSAGE        | Request   | message-id Header           | Local  | Connection | arbitrary
+| STOMP       | PRODUCER | ACK/NACK       | Response  | id Header                   | Local  | Connection | arbitrary
+| STOMP       | CONSUMER | MESSAGE        | Request   | message-id Header           | Remote | Connection | arbitrary
+| STOMP       | CONSUMER | ACK/NACK       | Response  | id Header                   | Remote | Connection | arbitrary
+
+### What are some examples of IDs that aren't message IDs?
+TODO: move some rows above into here
+
+### Isn't correlation ID the same as a message ID?
+A correlation ID is a system-wide lookup value that possibly can pass multiple steps. A message ID
+can have a scope as small as one segment (Ex. flow from producer to broker), and typically an
+implementation detail. It can bet the case that they are the same, but it is not commonly the case.
```

Comment:
you writing this from a hotel room in Vegas ?
```suggestion
implementation detail. It can be the case that they are the same, but it is not commonly the case.
```